packages/chrome-extension/scripts/github.com/repository.js

```diff
@@ -0,0 +1,21 @@
+function main() {
+  const exampleFolder = document.querySelector('[title="example"]');
+  const pubspecs = document.querySelectorAll('[title="pubspec.yaml"]');
```

Comment:
There can't be more than one right?

Reply:
I was wondering if there could be a case where there is more than one pubspec (maybe more than one folder displayed ?). The script would handle adding a Zapp button for each, but I can remove it if you find it weird 😅

Reply:
From my understanding, this looks for a file (not directory) called `pubspec.yaml` in the github file list (for the currently opened directory), which means only one file can be named that right? Or at least that's my assumption 😄

Reply:
Github adds for each row a `title="name"`, so it could be a directory or a file. But you are right, it doesn't makes sense to have multiple pubspec.yaml, Zapp wouldn't handle it.
I'll remove it :)

Reply:
Done :)

Reply:
The importer for GitHub looks for 2 paths relative to current directory; 

 - pubspec.yaml
 - lib/main.dart

Reply:
I cannot look for lib/main.dart without doing a network request (and adding more permissions to the chrome extension)

Reply:
That's fine, was just an FYI, a pubspec.yaml is good enough since the importer can check for the rest

Reply:
Oh okay I had misunderstood your message 👍